swagger/specification/applications/resource-manager/Applications.Core/preview/2023-10-01-preview/openapi.json

```diff
@@ -3002,7 +3002,7 @@
           "type": "object",
           "description": "environment",
           "additionalProperties": {
-            "type": "string"
+            "$ref": "#/definitions/EnvironmentVariableUpdate"
```

Comment:
@rynowak do you know what's making the `Update` suffix to be added in a couple of places?

Reply:
I'm not sure. 

I know that TSP **can** generate different schemas for `GET` and `PUT` operations. It looks like that's what's happening here, but I don't know why it's happening.